### PR TITLE
Fix some typos (found by codespell and typos)

### PR DESCRIPTION
**Pull request by stweil**
## Description

Related issue: #1001

## Motivation and Context
I noticed a typo while using the software.

## How has this been tested?
Manual review of the changes.

## Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## Checklist
- [x] I have read the [contributor guide](https://github.com/rdmorganiser/rdmo/blob/main/CONTRIBUTING.md).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: rdmo/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1173,7 +1173,7 @@ msgid "You are logged in as: "
 msgstr "Sie sind eingeloggt als: "
 
 #: core/templates/rest_framework_swagger/index.html:43
-msgid "Viewing as an anoymous user"
+msgid "Viewing as an anonymous user"
 msgstr "Ansicht als anoymer Benutzer"
```
Comment (Zack-83) [inline rdmo/locale/de/LC_MESSAGES/django.po:1177]:
```suggestion
msgstr "Ansicht als anonymer Benutzer"
```

**Review by Zack-83 — APPROVED** [review]

**Comment by MyPyDavid** [comment]
why was this merged on the `main` branch? Thought we have currently the `dev-2.2.0` for that...

**Comment by Zack-83** [comment]
My mistake :(

**Comment by MyPyDavid** [comment]
we need to check if the changes to translations in the migration files require new migrations.. 
and now the `dev-2.2.0` related branches need to be rebased to this new `main` ..

**Comment by jochenklar** [comment]
I think we should rebase `dev-2.2.0` after we merge most of the feature branches.

**Comment by jochenklar** [comment]
Migrations and translations seem ok. The changed migrations are old, and the translations were changed in the same way in the code and the `.po` files.

**Pull request merged**